action.js

```diff
@@ -0,0 +1,127 @@
+const mainmenu = document.querySelector('.mainMenu');
+const closemenu = document.querySelector('.closeMenu1');
+const openmenu = document.querySelector('.openMenu');
+const menuitems = document.querySelectorAll('nav .mainMenu li a');
+
+function show() {
+  mainmenu.style.display = 'block';
+  mainmenu.style.paddingTop = '80px';
+  mainmenu.style.top = '0';
+  mainmenu.style.marginBottom = '40px';
+  closemenu.style.display = 'inline';
+  closemenu.style.paddingLeft = '15%';
+}
+
+function close() {
+  mainmenu.style.top = '-180%';
+  mainmenu.style.display = 'flex';
+  mainmenu.style.paddingTop = '5px';
+  closemenu.style.display = 'none';
+}
+
+openmenu.addEventListener('click', show);
+closemenu.addEventListener('click', close);
+
+menuitems.forEach((item) => {
+  item.addEventListener('click', () => {
+    close();
+  });
+});
+
+const speakers = [
+  {
+    name: 'Yochai Benkler',
+    image: "<img src='images/sone.png' alt=''/>",
```

Comment:
- [ ] @MayPyone Great work implementing the changes requested, however, you forgot to add an alt message for these images, I would also urge the use of a more descriptive alt message eg `image of Yochai Benkler` instead of just `speaker` , this will make the interaction better for visually impaired.